Assets/ScaleMonk Ads/Editor/AdnetsWindow/AdsProvidersHelper.cs

```diff
@@ -16,8 +16,8 @@ namespace ScaleMonk.Ads
 {
     public class AdsProvidersHelper
     {
-        const string iosAdsVersion = "1.1.0";
-        const string androidAdsVersion = "1.0.23";
+        const string iosAdsVersion = "1.2.0";
+        const string androidAdsVersion = "1.1.0-rc.11";
```

Comment:
no subas esta version rc a master

Reply:
You can set android verison 1.1.0, it has been released yesterday

Reply:
Fixed